templates/accounts/set_password.html

```diff
@@ -44,7 +44,7 @@ <h1>Set a Password</h1>
 
 {% block scripts %}
 {% compress js %}
-<script src="{% static 'js/zxcvbn.js' %}"></script>
+<script src="{% static 'js/zxcvbn.min.js' %}"></script>
 <script src="{% static 'js/set-password.js' %}"></script>
 {% endcompress %}
```

Comment:
Should I move zxcvbn out of `compress js` ??

Reply:
No, `{% compress %}` bundles `zxcvbn` and `set-password` into a single file, which is what we want.

Reply:
Done 🙃